recipe/meta.yaml

```diff
@@ -10,6 +10,7 @@ source:
 
 build:
   number: 0
+  # trigger: 1 
```

Comment:
another issue here, while you are on it.  adjust test invocation of pip. then you can raise builder number too.  As you modified recipe, there will be an new hash valure anyway

Reply:
What is the recommended invocation?
Should it be this one?
{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv

Reply:
ok, that was for pip check